packages/orbit-components/src/hooks/useRandomId/index.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import { useUIDSeed, useUID } from "react-uid";
+
+import type { useRandomIdSeedType, useRandomIdType } from "./index.js.flow";
+
+export const useRandomIdSeed: useRandomIdSeedType = () => {
+  return useUIDSeed();
+};
+
+export const useRandomId: useRandomIdType = () => {
+  return useUID();
+};
```

Comment:
I recommend exporting `useRandomId` as default instead of named because the path is named `hooks/useRandomId` so people will expect it (I would). That would also send the correct message that `useRandomId` is the primary hook, and `useRandomIdSeed` is secondary, for more specific situations.